README.md

```diff
@@ -1,4 +1,33 @@
 # sycamore
 
+This is our project Front-End Capstone.
+
+# Setup & GitHub Flow
+
+1. Clone the repo from the organization to your computer
+```
+git clone https://github.com/sensational-sycamore/sycamore
+```
+2. Create a feature branch on your computer, and commit your work to it
+```
+git checkout main
+git pull origin main
+git checkout -b feature-branch-name
+git add .
+git commit
+```
+5. Push your work to a feature branch on your fork repo
+```
+git push origin feature-branch-name
+```
+6. Create a pull request from your feature branch to the main branch on the organization repo
+
+# Code Review
+
+1. Every PR should get code review and approval for all the team members prior to "Merge pull request"
```

Comment:
We should decide on how to split up the PR review work and who's responsible for reviewing what code.